glotaran/builtin/items/activation/gaussian.py

```diff
@@ -172,6 +173,8 @@ def parameters(
         scales = self.scale or [1.0] * nr_gaussians
         backsweep = self.backsweep is not None
         backsweep_period = self.backsweep if backsweep else 0
+        if isinstance(backsweep_period, Parameter):
```

Comment:
Adopted that suggestion.

Reply:
That said using a pydantic `Model` instead of `dataclass` for `GaussianActivationParameters` should also take proper care of type coercion.

Reply:
> That said using a pydantic Model instead of dataclass for GaussianActivationParameters should also take proper care of type coercion.

Since it is a drop in replacement, I have no objection to change it.

Reply:
> Tried to adopt that suggestion ...
> 
> But:
> 
> > glotaran\builtin\items\activation\gaussian.py:175: error: Argument 1 to "float" has incompatible type "Parameter | float | str | None"; expected "str | Buffer | SupportsFloat | SupportsIndex"  [arg-type]
> > glotaran\builtin\items\activation\gaussian.py:183: error: Unused "type: ignore" comment  [unused-ignore]

This is limitation of mypy, since it cannot understand that the type is narrowed done at that point. Either ignore it or add an assert that is of float instance.